semgrep-core/src/reporting/JSON_report.ml

```diff
@@ -217,9 +217,16 @@ let unsafe_match_to_match render_fix_opt (x : Pattern_match.t) : Out.core_match
     let* edit = render_fix x in
     Some edit.Textedit.replacement_text
   in
+  let file =
```

Comment:
This requires a comment here. You probably need to put the comment in your PR here so other people
can understand why we need those shenanigans.